openlibrary/templates/account/email/forgot-ia.html

```diff
@@ -36,7 +36,7 @@ <h2>$_('Forgot Your Internet Archive Email?')</h2>
       <div class="formElement">
         <div class="label"></div>
         <div class="input smaller padtop">
-	  $:_('<a href="https://archive.org/account/login.forgotpw.php">Forgot Archive.org Password</a>?')
+	  '<a href="https://archive.org/account/login.forgotpw.php">$_('Forgot your Archive.org Password?')</a>
```

Comment:
XXX **blocker** should this leading ' be removed?

Reply:
ah, yes! sorry, good catch!